.cspell.yml

```
@@ -27,7 +28,7 @@ languageSettings:
   - languageId: markdown
     ignoreRegExpList:
       - CodeBlock
-dictionaries: [pt-br, companies, cpp, dotnet, golang, node, softwareTerms]
+dictionaries: [pt-br, es-es, companies, cpp, dotnet, golang, node, softwareTerms]
```

```suggestion
dictionaries: [pt-br, es, companies, cpp, dotnet, golang, node, softwareTerms]
```
I prefer to use just "es", code for the Spanish language. And not use the region

---

@chalin In the URL it'll show as just `es`, so user-facing wise it'll be just `es`. In the code, as @chalin explained, it's a bit more complicated and we should define a variant. Right Patrice?

---

@theletterf @chalin I think we can use the generic dictionary of `es` to avoid region limitations and language variant conflicts. The `es` dictionary is valid and available on **cspell**:

https://github.com/streetsidesoftware/cspell/blame/main/packages/cspell-trie-lib/src/lib/models/locale/knownLocales.ts#L197

At the official Kubernetes documentation, we use plain `es` dictionary, and many Spanish contributors from different regions can collaborate without region word conflict.

---

As I explained in https://github.com/open-telemetry/opentelemetry.io/issues/4887#issuecomment-2273902715, IMHO, there is no such thing as a generic `es` dictionary, only the `es-ES` one. But correct me if I'm wrong.

---

As the `es-ES` dictionary is the only existing one, and as all words in that dictionary apply to all variants when it comes to academic and scientific language, I've decided to keep it. We can always improve this later.